src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/TlsTests.cs

```diff
@@ -44,10 +45,7 @@ public async Task TlsHandshakeRejectsTlsLessThan12()
             listenOptions =>
             {
                 listenOptions.Protocols = HttpProtocols.Http2;
-                listenOptions.UseHttps(_x509Certificate2, httpsOptions =>
-                {
-                    httpsOptions.SslProtocols = SslProtocols.Tls11 | SslProtocols.Tls12;
-                });
+                listenOptions.UseHttps(_x509Certificate2);
```

Comment:
This test isn't actually testing the right thing anymore. The goal is to test that Tls11 isn't allowed for HTTP/2. Revert this part and skip the test on Win8 instead.

You can't tell that the test is still broken because of the QuarantinedTest attribute.

Reply:
This should be skipped on Win8, but we disallow Tls11 with HTTP/2 even if we don't specify specific SslProtocols, right? I don't see what that part had to be reverted, but I don't think it matters one way or another.

Reply:
What are the default protocols on Win10?

Reply:
I'm not sure about all the allowed protocols, but according to https://docs.microsoft.com/en-us/dotnet/framework/network-programming/tls#configuring-schannel-protocols-in-the-windows-registry, TLS 1.2 is enabled by default on Win10/2016. Even Win8/2012 enable TLS 1.2 by default, there's just no ALPN support.

Reply:
Fair enough. We could probably use the default protocols here, but it would be unrelated to fixing the test issue (win8 ciphers).